### PR TITLE
change single letter internal variables #1572

### PR DESCRIPTION
**Pull request by tinumide**

### CODE REVIEW

**Review by bsipocz — COMMENTED** [review]
I confirm that remote tests pass with this (one exception, that is already failing on master)

One minor comment to address, and then this is good to get merged.

File: astroquery/simbad/tests/test_simbad.py [hunk 1]
```diff
@@ -113,9 +113,9 @@ def test_get_frame_coordinates(coordinates, expected_frame):
     actual_frame = simbad.core._get_frame_coords(coordinates)[2]
     assert actual_frame == expected_frame
     if actual_frame == 'GAL':
-        l, b = simbad.core._get_frame_coords(coordinates)[:2]
-        np.testing.assert_almost_equal(float(l) % 360, -67.02084 % 360)
-        np.testing.assert_almost_equal(float(b), -29.75447)
+        length, breadth = simbad.core._get_frame_coords(coordinates)[:2]
+        np.testing.assert_almost_equal(float(length) % 360, -67.02084 % 360)
+        np.testing.assert_almost_equal(float(breadth), -29.75447)
```
[the remark this answers is not in the document]
Comment (tinumide) [reply astroquery/simbad/tests/test_simbad.py:118]: Oh I see. Funny how I just assumed it should be length and breadth 😂😂😂. Thanks @bsipocz

[the remark this answers is not in the document]
Comment (bsipocz) [reply astroquery/simbad/tests/test_simbad.py]: no worries, this is jargon that we should not have relied on anyway.

**Review by bsipocz — APPROVED** [review]

**Comment by bsipocz** [comment]
Thanks @tinumide!

**Pull request merged**